rest/route/annotations.go

```diff
@@ -323,6 +323,125 @@ func (h *annotationByTaskPutHandler) Run(ctx context.Context) gimlet.Responder {
 	return responder
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// PATCH /rest/v2/tasks/{task_id}/annotation
+
+type annotationByTaskPatchHandler struct {
+	taskId string
+
+	user       gimlet.User
+	annotation *restModel.APITaskAnnotation
+}
+
+func makePatchAnnotationsByTask() gimlet.RouteHandler {
+	return &annotationByTaskPutHandler{}
+}
+
+func (h *annotationByTaskPatchHandler) Factory() gimlet.RouteHandler {
+	return &annotationByTaskPutHandler{}
+}
+
+func (h *annotationByTaskPatchHandler) Parse(ctx context.Context, r *http.Request) error {
+	var err error
+	h.taskId = gimlet.GetVars(r)["task_id"]
+	taskExecutionsAsString := r.URL.Query().Get("execution")
```

Comment:
This looks a lot like Parse function on annotationByTaskPutHandler - can we modularize these a bit?

Reply:
created a helper fuction